pkg/controller/metrics/adapters/metrics_sender_to_kafka.go

```diff
@@ -0,0 +1,71 @@
+package adapters
+
+import (
+	"context"
+	log "github.com/golang/glog"
+	"github.com/opensds/opensds/pkg/model"
+	"github.com/segmentio/kafka-go"
```

Comment:
Considering you import this package, please specify the version in `Gopkg.toml` file.

Reply:
Can you please suggest on how to find the version of this dependency ? "github.com/segmentio/kafka-go"

Reply:
I would suggest choosing the latest published version, you can find it on: https://github.com/segmentio/kafka-go/releases, but you can specify the latest commit id if there is no release.

Reply:
thanks. updated the .toml file.